teps/0031-tekton-bundles-cli.md

```diff
@@ -66,3 +71,83 @@ Provide a way for users to...
 * Users may publish to any repository they have access to.
 
 * Only valid Tekton Bundles are produced.
+
+## Proposal
+
+Amend the Tekton CLI to provide the following commands (more detail later):
+
+```shell
+# REF = OCI reference, eg. docker.io/myworkspace/mybundle:1.0
+# BUNDLE_OBJECT = path to a file or directory or STDIN of a valid Tekton Task or Pipeline object to be included in the
```

Comment:
In case of STDIN, I don't think we should limit to a single task / pipeline.
I'd still like to be able to `kustomized build <folder> | tkn bundle push <REF> -`

Reply:
Definitely. The intention was to parse all objects out of the YAML so I will make that clear.